doc/grass_database.html

```diff
@@ -52,29 +53,31 @@ <h3>GRASS GIS Database</h3>
 inside in a specific format and is accessible by specific tools.
 GRASS GIS Databases is in GRASS GIS often called GISDBASE or DATABASE.
 
-<h3>GRASS Locations</h3>
+<h3>GRASS Projects</h3>
 
-Location is a directory which contains GRASS Mapsets which are its subdirectories.
-All data in one Location have the same projection (coordinate system, datum).
-Each Location must contain Mapset called PERMANENT.
-Typically, a Location contains all data related to one project
+Project is further organized into mapsets which are subdirectories of
```

Comment:
```suggestion
A Project is further organized into mapsets which are subdirectories of
```

(I am not a native speaker but an "A" missing here would sound unusual to me)